### DISCUSSION TITLE
Server connected once while creating a template, quit PT, reopened and now it won't connect.

### DISCUSSION THREAD

**Discussion by brendanreza** [apohl79/audiogridder#657] [Help]
MacOS 10.14.  Using v1.1.1

Not using a separate server computer.  Just running the AudioGridder Server on my iMac.

I was able to connect the AGridder AAX Plugin to the AudioGridder Server after installing.  I setup a template in Pro Tools.  I then quit PT and now when I reopen, the AGridder AAX plugin doesn't connect to the AudioGridder Service app on my computer.

Nothing has changed.  I've tried wiping cache and rescanning, renaming the server, etc.  Nothing is working.  I get the attached.  What am I doing wrong?
![Screen Shot 2021-07-21 at 7 46 59 PM](https://user-images.githubusercontent.com/11874379/126576886-66a0df59-0d8a-4631-99fe-ec4c916fd950.png)

**Comment by UncoveredPitch** [comment]
Not the dev but here's a few ideas from my own struggles until someone more knowledgeable shows up here:

1. It doesn't look like the AudioGridder Server app is active. You should otherwise see another similar icon next to the one on your screenshot. So re-start the Server app from the dock or applications folder. Maybe the app has crashed in the background without you noticing.
2. On your screenshot it says 1.2.0 beta4 whereas you say you're using v 1.1.1.. Maybe you're mixing and matching?
3. Wiping the cache and rescanning, or renaming the server has never done anything for me. 
4. Re-starting the Server app, re-starting the DAW and if need be re-starting the Mac has however re-established the connection on many occasions. Now, before using AudioGridder, I always re-start the Server app, then the DAW. I never leave a session open "overnight" and then wake up the Mac from sleep like I used to do before using AudioGridder.

Good luck!

**Comment by brendanreza** [comment]
Thanks for the input!  Yeah I've since gotten familiar with both apps.  I
was trying both b1.2.4 and the shipping 1.1.1.  Both do the same thing,
sorry for the confusion though.

I read that most people have more success using this as intended (with an
actual server computer).  But in my eyes, there should be no reason why
this shouldn't work.

My server and plugin tray are running before my DAW opens.  When my DAW
opens, the plugin is loaded in the project.  The AudioGrid Plugin tray
shows my track names where the AGridder plugin exists in the project.  My
server just refuses to connect anymore, regardless if I rename it or
restart the computer.

I understand this is a free app, so I'm not complaining!  I'm just
thoroughly confused, as I had it working great for about 20 minutes, setup
the template, and now it refuses to work at all.

**Reply by UncoveredPitch** [reply]
You're welcome. Looks like you need the dev to chip in here. I can confirm, however, that using a local server(on one machine) works perfectly well. Actually in my own testing better than using my second Mac as a server computer! I'm still not sure why you can't see the second AudioGridder icon though. They should look like yin and yang next to each other when both the plug-in tray and server app are running!

**Comment by brendanreza** [comment]
That's great to hear!  To clarify...I *do* see both icons.  The Plugin tray
app just appears and disappears randomly.

The main question is why I can't connect to the server even though
everything is running and the AGridder plugin and the Plugin Tray
application are talking to each other.  And why was it working for me
before restarting my computer/DAW?  Hopefully it's something stupid on my
end.

**Reply by MrPisquan** [reply]
a) Do you have your computer connected to a network? AudioGridder needs an IP address, I think a network connection is required. For example, I am not able to connect with AG when my WIFI is disabled.

b) Try to connect with the 127.0.0.1 server address
https://github.com/apohl79/audiogridder/issues/113

I really hope it could help.

**Comment by brendanreza** [comment]
Thanks for the tip.

My computer is connected to a wireless network.  My computer's IP address
starts with 192.

You mentioned trying a 127 address.  How would I change that?

**Comment by MrPisquan** [comment]
Hi,

When you choose the server from the plugin try to digit the 127.0.0.1 address instead. 
There was a link about this in my previous answer, go check it.
Also, in the "yin" (white) audiogridder app you may try to "connect all to" option. You can choose there the server to be connected to.
Sorry I have not my computer here and my memory could not be that precise, but you have an idea.

Good luck!

**Reply by MrPisquan** [reply]
Hi,
for chance I incurred in a similar problem.
here's how I sorted it out (pasted from my recent post) :
"Maybe could be useful for someone, I solved the problem installing a virtual network adapter.
I added the  Microsoft Loopback Adapter from Device manager (Windows), than assigned the adapter a static IP.
Now the plugin reconnects with the server with no problems.
I had the suspect that there was something related to the network when I noticed that AG worked fine just when another computer is connected via wired LAN while it makes no difference whether there is or there isn't a WI-FI connection. 
Actually, the problem I referred came in place with no network connections (I had the other computers disconnected).
I hope someone would find this solution helpful."
